src/runtime/ranger/ranger_resource_policy_manager.cpp

```diff
@@ -599,34 +586,50 @@ dsn::error_code ranger_resource_policy_manager::sync_policies_to_app_envs()
         req->__set_app_name(app.app_name);
         req->__set_keys(
             {dsn::replication::replica_envs::REPLICA_ACCESS_CONTROLLER_RANGER_POLICIES});
-        bool is_policy_matched = false;
+        std::vector<matched_database_table_policy> matched_database_table_policies;
         for (const auto &policy : table_policies->second) {
             // If this table does not match any database, its Ranger policies will be cleaned up.
             if (policy.database_names.count(database_name) == 0 &&
                 policy.database_names.count("*") == 0) {
                 continue;
             }
+            // If this table does not match any database table, its Ranger policies will be cleaned
+            // up.
```

Comment:
The added comments don't match the line below, the line just skip to update `matched_database_table_policies`.